charts/fpl-case-service/values.yaml

```diff
@@ -5,20 +5,49 @@ tags:
 java:
   image: hmctspublic.azurecr.io/fpl/case-service:latest
   environment:
+    AUTH_IDAM_CLIENT_BASEURL: https://idam-api.aat.platform.hmcts.net
```

Comment:
we can also use `{{ .Values.global.environment }}` instead of hardcoding the environment name